pkg/ccl/changefeedccl/encoder_protobuf.go

```diff
@@ -277,29 +280,26 @@ func (e *protobufEncoder) buildMetadata(
 	return meta, nil
 }
 
-// encodeRowToRecord converts a Row into a Record proto.
-func encodeRowToRecord(row cdcevent.Row) (*changefeedpb.Record, error) {
+func encodeRowToValues(row cdcevent.Row, values *map[string]*changefeedpb.Value) error {
```

Comment:
You shouldn't need to pass a pointer to a map; maps in go are passed by reference

Example: https://go.dev/play/p/unFbnLaLal1